testing/endpoint.go

```diff
@@ -431,7 +431,8 @@ func (endpoint *Endpoint) SendPacket(
 	channelCap := endpoint.Chain.GetChannelCapability(endpoint.ChannelConfig.PortID, endpoint.ChannelID)
```

Comment:
Adding here now so I don't forget about it: there's also `SendPacket` in `endpoint` which can be updated to match `SendPacket` in `packet.go`.